scripts/docker/pom-modify/PomFile.java

```diff
@@ -83,6 +86,35 @@ private void parseFields(String pom) {
             dBuilder = dbFactory.newDocumentBuilder();
             Document doc = dBuilder.parse(pomFile);
 
+	    // If pom file already patched, stop right here
```

Comment:
Shouldn't the check also see if the version is the same or not and still patch the POM file if the versions mismatch?

Reply:
Not sure. What would happen if we specify dependencies on two versions of the plugin?

Reply:
Here's another issue I need to check: I have not really tested the 'false' side of the logic and I also haven't specified a default for the configuration. Let me check the behavior before we proceed with this PR.

Reply:
I don't foresee a use case where one would be dependent on two different versions of iDFlakies. However, if your change is to ease the development and testing of iDFlakies, it may be beneficial to enhance your code here to stop only if the two versions are the same.

Reply:
I think what would happen if you declare dependencies on two versions of the plugin is that it would take only use one of the versions, but there is no guarantee as to which one it would use. Best practice would be to make sure this is just one version of the same plugin declared.

Reply:
That's what the change does right now: It prevents the repeated addition of the same plugin irrespective of the version. It may be good to issue a warning, though, to inform the developer, e.g., if heshe actually hopes to overwrite an older version with a newer one. If you're ok with that, I'll add it later today.